lib/glific/third_party/bigquery/bigquery_worker.ex

```diff
@@ -76,6 +77,7 @@ defmodule Glific.BigQuery.BigQueryWorker do
       make_job_to_remove_duplicate("messages_media", organization_id)
       make_job_to_remove_duplicate("flow_contexts", organization_id)
       make_job_to_remove_duplicate("profiles", organization_id)
+      make_job_to_remove_duplicate("contact_histories", organization_id)
```

Comment:
We don't need to update the data in contact history so there is no way of duplicate data. Let's remove this.